lib/utils.nix

```diff
@@ -2,33 +2,28 @@
 
 {
   config.lib.stylix = {
```

Comment:
We should probably create a library independent of config and tell the user how to extend their lib with ours.

This way we can finally be free of the config argument.

Reply:
A completely separate library (`stylix` / `stylixLib` or something) may be easier than trying to extend the existing `lib`.

Reply:
I think a separate library is probably a better idea, extending lib doesnt seem to be worth the effort